src/renderer/renderer.cpp

```diff
@@ -188,7 +184,7 @@ vec4 applyWaterEffect(vec4 color) {
     }
   }
 
-  int adjustedIndex = (index & 0x3) | 0x8;
+  const int adjustedIndex = 0x8;//(index & 0x3) | 0x8;
```

Comment:
I thought about this a bit, and I realized we can implement the same operation without using bit-wise operators like so:

```
int adjustedIndex = int(mod(float(index), 4) + 8);
```

That should be compatible with GL ES, and produce the same outcome.

Reply:
awesome

Reply:
I've made that change in #533